Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -59,6 +59,10 @@
         <Parent guid="guidNodeToolsWseContextMenuCmdSet" id="idmNpm"/>
       </Group>
 
+      <Group guid="guidMigrateToJspsCmdSet" id="cmdidJspsMigrate">
+        <Parent guid="guidSHLMainMenu" id="IDM_VS_CTXT_PROJNODE"/>
```

Comment:
Should there be another entry for the revert command?

Reply:
I don't think a separate Group is necessary